optuna/distributions.py

```diff
@@ -10,6 +10,10 @@
 
 
 CategoricalChoiceType = Union[None, bool, int, float, str]
+_int_distribution_deprecated_msg = "Use :class:`~optuna.distributions.IntDistribution` instead."
```

Comment:
[imo] I think variables should be introduced with references, so how about adding them in a subsequent pull request?

Reply:
Sure, just thought someone might be confused why similar classes are there but I agree that we can add it in a subsequent pull request.